### PR TITLE
Redux counter

### PR DESCRIPTION
**Pull request by ogwurujohnson**

### CODE REVIEW

File: src/components/Counter.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
 import React, { Component } from "react";
 import { connect } from 'react-redux';
-import { increment, decrement } from '../actions';
+import { increment, decrement,incrementOdd, asyncIncrement } from '../actions';
 
 class Counter extends Component {
     incrementIfOdd = () => {
         // Stretch Problem: Implement an increment function that
         // only increments if the counter value is odd
+        return this.props.incrementOdd();
```
Comment (mattshardman) [inline src/components/Counter.js]: there is probably no point having this separate function if all it does is call the incrementOdd action

File: src/components/Counter.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
     };
 
     incrementAsync = () => {
         // Stretch Problem: Implement an increment function that
         // increments after waiting for one second
+        setInterval(() => {
+            this.props.asyncIncrement()
+        }, 1000)
```
Comment (mattshardman) [inline src/components/Counter.js]: You might as well just call ```increment``` if the setInterval logic is in this file

**Comment by mattshardman** [comment]
Good job Johnson you've set up redux correctly and completed the stretch challenges - great work!

File: src/components/Counter.js (hunk 1 above)
Comment (ogwurujohnson) [reply src/components/Counter.js]: yeah, true. figured that out yesterday. Thanks for pointing it out, would make the change

**Pull request merged**